x/evm/types/msg_add_chain.go

```diff
@@ -42,6 +42,7 @@ func (m AddChainRequest) ValidateBasic() error {
 		NativeAsset:           m.NativeAsset,
 		SupportsForeignAssets: true,
 		KeyType:               m.KeyType,
+		Module:                "evm", // cannot use constant due to import cycle
```

Comment:
```suggestion
		Module:                ModuleName
```

Reply:
but more to that, why is this `chain` variable even created and validated in the first place in this message's `Validate` method?

Reply:
Because later on we will be adding a `exported.Chain` struct to the keeper based on the fields of this message (with the confirm-chain command). So instead of validating each field separately, we just assemble the struct and use its method.